core/capabilities.go

```diff
@@ -117,6 +118,7 @@ var CapabilityNameLookup = map[Capability]string{
 	Capability_AudioToText:                "Audio to text",
 	Capability_SegmentAnything2:           "Segment anything 2",
 	Capability_LLM:                        "Large language model",
+	Capability_TextToSpeech:               "Text to speech",
```

Comment:
Should we call this TextToAudio to make it more generatic and match up with the other naming?

Reply:
@rickstaa and I had a discussion about this previously and ended up on Text to Speech for two primary reasons:

1) The functionality implemented has been referred to as text-to-speech for a long time and is immediately clear to most people what the expected output will be
2) TextToAudio would better naming fit for a pipeline for [stable audio](https://huggingface.co/stabilityai/stable-audio-open-1.0) and other music-gen models

From a more architectural standpoint while at the moment it requires more boilerplate in go-bindings and in go-livepeer, it seems to me that it is more ideal to implement isolated and functionality specific pipelines - That is, more endpoints being preferable to switching implementations in the routes/pipelines based on matching one or more model_ids.

Sometimes it is hard to know where to draw the line, but I'm leaning towards the qualifications for an endpoint to share use of multiple models to have at least one of the following:
* nearly identical input schema
* ability to interchange models without significant implementation differences and dependencies
* the differences in output is more subjective and or qualitative, and less use-case specific.

Sorry to be long winded here, but happy to consider alternative perspectives.